packages/medusa/src/services/order-edit.ts

```diff
@@ -388,6 +411,85 @@ export default class OrderEditService extends TransactionBaseService {
     return orderEdit
   }
 
+  async addLineItem(orderEditId: string, data: AddOrderEditLineItemInput) {
+    return await this.atomicPhase_(async (manager) => {
+      const lineItemServiceTx = this.lineItemService_.withTransaction(manager)
+
+      const orderEdit = await this.retrieve(orderEditId, {
+        relations: [
+          "order",
+
+          "order.cart",
+          "order.cart.customer",
+          "order.cart.discounts",
+          "order.cart.discounts.rule",
+          "order.cart.gift_cards",
+          "order.cart.region",
+          "order.cart.region.tax_rates",
+          "order.cart.shipping_address",
+          "order.cart.shipping_methods",
+
+          "order.region",
+        ],
+      })
+
+      if (!OrderEditService.isOrderEditActive(orderEdit)) {
+        throw new MedusaError(
+          MedusaError.Types.NOT_ALLOWED,
+          `Can not add an item to the edit with status ${orderEdit.status}`
```

Comment:
suggestion: `Can not add an item on the order edit ${orderEditId} with the status ${orderEdit.status}`